shacl12-overview/index.html

```diff
@@ -373,61 +373,211 @@ <h2>Introduction</h2>
 
 		<section id="whatsnew">
 			<h2>What's New in SHACL 1.2</h2>
-			<p class="todo">
-				The main information about SHACL 1.0 &rarr; 1.2 will be added here.
-			</p>
-			<p>
-				SHACL 1.0 Core <a href="https://www.w3.org/TR/2017/REC-shacl-20170720/#shacl-shacl">included an appendix</a> providing a shapes graph to be used to validate shapes graphs according to the SHACL 1.0 Core specification. SHACL 1.2 has moved this information from the Core specification to an appendix in this Overview to allow for information relevant to parts of SHACL 1.2 not in the Core. A note about this new SHACL-SHACL location is retained in SHACL 1.2 Core.
-			</p>
+      <p>
+        SHACL 1.2 caters for <a href="https://www.w3.org/TR/rdf12-new/">new developments in RDF 1.2 and SPARQL 1.2</a>, formalises versions of original SHACL documents published only as Notes and takes SHACL into new areas such as rules for inferencing and profile defining.
```

Comment:
W3C documents typically use US English (e.g., `formalizes`). UK English (e.g., `formalises`) is _sometimes_ used, in which case each document should be internally consistent.
```suggestion
        SHACL 1.2 covers <a href="https://www.w3.org/TR/rdf12-new/">new developments in RDF 1.2 and SPARQL 1.2</a>, formalizes versions of original SHACL documents published only as Notes, and takes SHACL into new areas such as rules for inferencing and profile definition.
```